bad_connector_woocommerce/models/product/common.py

```diff
@@ -94,6 +247,7 @@ class WooProductProductAdapter(Component):
     _inherit = "woo.adapter"
     _apply_on = "woo.product.product"
     _woo_model = "products"
+    _woo_product_variation = "products/{product_id}"
```

Comment:
This is beneficial in the search method since it only becomes active when the model is 'woo.product.product,' as we are passing it in kwargs, and the '_woo_product_variation' enables us to make API calls for variations. If we use '_woo_data,' it requires manual modification of the resource path.